src/engine/DisjunctionConstraint.cpp

```diff
@@ -127,17 +150,35 @@ List<PiecewiseLinearConstraint::Fix> DisjunctionConstraint::getSmartFixes( ITabl
 
 List<PiecewiseLinearCaseSplit> DisjunctionConstraint::getCaseSplits() const
 {
-    return _disjuncts;
+    return List<PiecewiseLinearCaseSplit>( _disjuncts.begin(), _disjuncts.end() );
```

Comment:
What's the difference between the line erased and the new line?

Reply:
Return type _disjuncts is a Vector now. This method will become obsolete in the long run.